Makefile

```diff
@@ -1,14 +1,73 @@
+SHELL := /bin/bash
+
+NAME ?= terrahelp
+
 BUILDARGS ?= -mod=vendor
 
+BIN := $(CURDIR)/bin
+DIST := $(CURDIR)/dist
+OUPUT_FILES := $(BIN) $(DIST)
+
+PLATFORMS ?= darwin linux
+ARCH ?= amd64
+OS = $(word 1, $@)
+
 check:
 	go vet $(BUILDARGS) ./...
+.PHONY: check
 
 test:
 	go test $(BUILDARGS) -v ./...
+.PHONY: test
 
 build:
```

Comment:
should build not have a prerequisite with dependencies?

Reply:
The main reason dependencies target was not a prerequisite of the other targets in the file was because `go build` and the other commands are setup to use the vendor directory.  While modules is still new, but official now, we are still using the vendor directory.

Reply:
Jonathon and I have chatted about this and where appropriate the new prerequisites stand, but dependencies is not there so we avoid unnecessary downloads during builds.